src/System.Runtime.CompilerServices.Unsafe/ver/Private.Versioning.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0</TargetFrameworks>
+  </PropertyGroup>

```

Comment:
We either need to add the version properties here, or share them in a props file.